src/main/webapp/WEB-INF/themes/mobile/resource/css/peer-review.css

```diff
@@ -106,3 +106,15 @@
   font-weight: 600;
   text-decoration: underline;
 }
+
+span.underline {
+  text-decoration: underline;
+}
+
+span.strike {
+  text-decoration: line-through;
+}
+
+.monospace {
+  font-family: $font-face-monospace;
```

Comment:
This is a CSS file, not SCSS, so this variable won't get transformed.  
Replace with:
`font-family: "Courier New", Courier, monospace;`

Reply:
ah, thanks for the catch!

Reply:
@samV-plos I just moved these styles to base.css, assuming we want them on all mobile pages?

Reply:
yes, that's great, thanks!